### PR TITLE
Add protein-translation exercise

### PR DESCRIPTION
**Pull request by ricemery**
New Pull Request for the protein-translation exercise.

* Removed Option from the method signature.
* Lower difficulty to 3 and removed Optional Value from topics.

### CODE REVIEW

File: exercises/protein-translation/src/test/scala/ProteinTranslationTest.scala [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  test("Identifies tyrosine codons") {
+    pending
+    ProteinTranslation.translate("UAU") should be(Seq("Tyrosine"))
+    ProteinTranslation.translate("UAC") should be(Seq("Tyrosine"))
+  }
+
+  test("Identifies cysteine codons") {
+    pending
+    ProteinTranslation.translate("UGU") should be(Seq("Cysteine"))
+    ProteinTranslation.translate("UGU") should be(Seq("Cysteine"))
```
Comment (ErikSchierboom) [inline exercises/protein-translation/src/test/scala/ProteinTranslationTest.scala]: This line is the same as the line above it.

File: exercises/protein-translation/src/test/scala/ProteinTranslationTest.scala [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  }
+
+  test("Identifies tryptophan codons") {
+    pending
+    ProteinTranslation.translate("UGG") should be(Seq("Tryptophan"))
+    ProteinTranslation.translate("UGG") should be(Seq("Tryptophan"))
```
Comment (ErikSchierboom) [inline exercises/protein-translation/src/test/scala/ProteinTranslationTest.scala]: This line is the same as the line above it.

File: exercises/protein-translation/src/test/scala/ProteinTranslationTest.scala (hunk 1 above)
Comment (ricemery) [inline exercises/protein-translation/src/test/scala/ProteinTranslationTest.scala]: Good catch!

**Comment by ricemery** [comment]
Thanks @ErikSchierboom  - I fixed the duplicate assertions in the test suite.

**Comment by abo64** [comment]
Thank you @ricemery

**Pull request merged**